stdlib/asm/crypto/hashes/native.masm

```diff
@@ -89,7 +89,7 @@ export.hash_memory
     dup.13 mem_loadw
 
     # set the padding (9 cycles)
-    swapw dropw push.1.0.0.0
+    swapw dropw push.0.0.0.0
```

Comment:
nit: instead of `push.0.0.0.0` we could use `padw` (won't change the cycle count, but is a bit cleaner to write).

Also, could we simplify this like so:
```
dropw dup.9 mem_loadw padw
```

Reply:
Done